game_engine/main.cpp

```diff
@@ -245,9 +252,11 @@ int main(int argc, char *argv[]) {
             results["terminated"][to_string(player_id)] = player.terminated;
         }
     }
-
+	
+    results["execution_time"] = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now() - engine_start).count();
```

Comment:
I would like to see this added to the `game_statistics`. That way it will be recorded in the replay file and server database.

The server will just ignore (and drop) it in the `results` from the engine. If it's useful in `results` for local testing I'm fine with leaving it there as well.

It would also be nice to get total time and maximum turn time used for each bot (we had one or both of those in h2). But that isn't trivial and doesn't need to be done in this PR.